src/app/board/[boardId]/error.js

```
@@ -0,0 +1,24 @@
+"use client" // Error boundaries must be Client Components
+
+import { useEffect } from "react"
+
+export default function Error({ error, reset }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)